styles/card.css

```diff
@@ -1,62 +1,87 @@
-@import url('/styles/main.css');
-.card {
-    display: flex;
-    flex-direction: column;
-    /* background: purple; */
-    padding: 0.8rem;
-    width: auto;
-    max-width: 20rem;
-    /* height: auto; */
-    height: max-content;
-    border-radius: 0.2rem;
-    /* border: 1px solid black; */
-    box-shadow: 8px -4px 35px -2px rgba(0, 0, 0, 0.40);
-    transition: all 0.3s;
-}
-
-.card:hover {
-    box-shadow: 16px -8px 70px -4px rgba(0, 0, 0, 0.77);
-}
-
-.horizontal-img {
-    /* height: 25;
-    width: 12vw; */
-    margin: auto;
-    height: 12rem;
-    padding: 0.5rem;
-}
-
-.horizontal-card {
-    border: 1px solid black;
-    width: 25rem;
-    height: 12rem;
-}
-
-.body-h {
-    padding: 1rem;
-    position: relative;
-    bottom: 13rem;
-    left: 8rem;
-    float: right;
-}
-
-.card-img {
-    align-items: center;
-    justify-content: center;
-    margin: auto;
-    width: 100%;
-    max-width: 15rem;
-    height: auto;
-    padding: 0.5rem;
-}
-
-.card-buttons {
-    float: right;
-}
-
-.card-text-overlay {
-    position: relative;
-    bottom: 5rem;
-    left: 5rem;
-    color: whitesmoke;
-}
\ No newline at end of file
+    @import url('/styles/main.css');
+    .card {
+        display: flex;
+        flex-direction: column;
+        padding: 0.8rem;
+        width: auto;
+        max-width: 20rem;
+        height: max-content;
+        border-radius: 0.2rem;
+        box-shadow: 8px -4px 35px -2px rgba(0, 0, 0, 0.40);
+        transition: all 0.3s;
+    }
+    
+    .card:hover {
+        box-shadow: 16px -8px 70px -4px rgba(0, 0, 0, 0.77);
+    }
+    
+    .horizontal-img {
+        margin: auto;
+        height: 12rem;
+        padding: 0.5rem;
+    }
+    
+    .horizontal-card {
+        border: 1px solid black;
+        width: 25rem;
+        height: 12rem;
+    }
+    
+    .body-h {
+        padding: 1rem;
+        position: relative;
+        bottom: 13rem;
+        left: 8rem;
+        float: right;
+    }
+    
+    .card-img {
+        align-items: center;
+        justify-content: center;
+        margin: auto;
+        width: 100%;
+        max-width: 15rem;
+        height: auto;
+        padding: 0.5rem;
+    }
+    
+    .card-buttons {
+        float: right;
+    }
+    
+    .text-only {
+        position: relative;
+        padding: 1rem;
+        align-items: center;
+        height: auto;
+        border: 1px solid black;
+    }
+    
+    .close {
```

Comment:
Could have a small addition here, close:hover might have a small background color change , and the close element might want a round bg which sort of highlights the hover.